contents/projectManagement/githubActions.md

```diff
@@ -0,0 +1,162 @@
+<frontmatter>
+  title: Workflow Automation with Github Actions
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Github Actions
+
+📝 [@alyip98](https://github.com/alyip98)
+
+🔎 
+
+
+## What is Github Actions?
+<!--Explain how X fits into the big picture of SE. Describe it relative to topics the reader is likely to know.-->
+Github Actions is a workflow automation solution that is tightly integrated into the Github ecosystem. It is commonly used to provide end-to-end continuous integration and continuous deployment (CI/CD) directly in your repository. With its tight integration with Github's ecosystem of services, users have the potential to automate more than just CI/CD.
+
+---
+
+## Why Github Actions?
+<!--Motivate the reader to learn about X. Describe benefits of X to make the reader interested in X. Try to give a balanced view of by mentioning also WHY NOT X i.e., mention both advantages and disadvantages.-->
+### Easy to use
+Setting up a workflow in a repository is easy and hassle-free. Templates for popular languages are available and the built-in editor offers code completion and documentation at a glance.
+![](https://i.imgur.com/JiVXlO4.png)
+
+
+
+### Reusable actions
+Actions are a small, reusable unit in a workflow. When setting up a workflow, we can use actions written by the community. The marketplace allows users to browse actions written by others for use in our workflows.
+
+
+![](https://i.imgur.com/m9PddIu.png)
+
+
+
+### Cross OS support
+Github Actions gives you the option to [run workflows](https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources) on Ubuntu, Windows and macOS.
+
+### Docker support
+Github Actions provides the ability to run docker images for projects that use a dockerized environment.
+
+### Powerful
+Integration with the Github ecosystem allows Github Actions to do more than just build/test/deploy. You can automate everything related to your Github project, including bug triaging, labeling, automated dependency updating etc.
+
+### Affordable
+The free plan offers 2000 minutes of build time per month for public repositories and a pay-per-minute plan for further use.
+
+---
+
+## How does Github Actions work
```

Comment:
Since these benefits' descriptions are relatviely short, probably consider using numbered list instead of titles?
e.g. `1. **Affordable**: ...`